packages/react-native-editor/bin/build_e2e_ios_app

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash -eu
 
+set -o pipefail
```

Comment:
I considered adding this in the code path where the output is piped, but it seemed better to set it as an option for the whole script instead.

Reply:
Also, I didn't make it part of the shebang because of the following ShellCheck warning: On most OS, shebangs can only specify a single parameter. [SC2096]